__tests__/hooks/useGroupInfo.test.js

```diff
@@ -686,16 +657,6 @@ describe('useGroupInfo should run correctly', () => {
     expect(spyAlert).toHaveBeenCalled();
   });
 
-  it('leaveGroup should run correctly', async () => {
-    const {result} = renderHook(() => useGroupInfo(), {wrapper});
-    await result.current.leaveGroup();
-    expect(mockRemoveMember).toHaveBeenCalled();
-    expect(mockAddCreate).toHaveBeenCalled();
-    expect(mockedResetNavigation).toHaveBeenCalled();
-    expect(moctAddMemeber).toHaveBeenCalled();
-    expect(result.current.newParticipant).toEqual([]);
-  });
-
   it('onReportGroup should run correctly', async () => {
     const {result} = renderHook(() => useGroupInfo(), {wrapper});
     await result.current.onReportGroup();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The test for the `onReportGroup` function is incomplete. There are no assertions to check if the function behaves as expected. Consider adding relevant assertions.
```javascript
it('onReportGroup should run correctly', async () => {
  const {result} = renderHook(() => useGroupInfo(), {wrapper});
  await result.current.onReportGroup();

  // Add relevant assertions here
});
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->